lib/phoenix_live_view/html_algebra.ex

```diff
@@ -399,6 +399,9 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
       {:ok, string} when is_binary(string) ->
         ~s(#{attr}="#{string}")
 
+      {:ok, {atom, _, _}} when atom in [:<<>>, :<>] ->
```

Comment:
@josevalim @chrismccord not sure if there is a better way to identify that it is a concatenation. Let me know 🙌🏻